form.html

```diff
@@ -0,0 +1,220 @@
+        >
+        <br />
+        <textarea
+          type="text"
+          id="txt-area"
+          class="box"
+          name="letter"
+          maxlength="6000"
+          rows="10"
+          cols="100"
+          required
+          spellcheck="false"
+        ></textarea>
+        <br />
+        <button class="bttn-3" type="submit">Enviar para o futuro</button>
+      </form>
+      <p id="result"></p>
+      <p id="citation">
+        "Todos ansiamos por amor – todos o buscamos –, mesmo quando não temos
+        esperança de que ele possa ser de fato encontrado".
+      </p>
+      <p id="bell-hooks">- Bell Hooks</p>
+    </div>
+  </body>
+  <style>
+    @font-face {
+      font-family: SpecialElite;
+      src: url(./public/SpecialElite-Regular.ttf);
+    }
+
+    body {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    .flex-column {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    .statement {
+      font-family: SpecialElite;
+      font-size: 1.25rem;
+      line-height: 1.786;
+      word-spacing: 0.1rem;
+      text-align: center;
+      font-size: 0.938rem;
+      color: #dd4d19;
+      margin-top: 1.25rem;
+      text-shadow: 0.25rem 0.25rem 0.25rem #00000040;
+      display: flex;
+      flex-direction: column;
+    }
+
+    #txt-area.box {
+      margin-top: 2rem;
+      width: 13.875rem;
+      height: 7.805rem; /*aspec ratio is 270px x 151.888px */
+      border-radius: 0.313rem;
+      box-shadow: 0.25rem 0.25rem 0.25rem #dd4d19,
+        0.25rem 0.25rem 0.25rem #e27c57;
+      resize: vertical;
+      padding: 2%;
+      font-size: 1.2rem;
+      font-family: SpecialElite;
+      border: 0.2rem dashed #dd4d19;
+      letter-spacing: 0.2rem;
+      word-spacing: 0.4rem;
+      line-height: 1.4rem;
+      display: flex;
+      flex-direction: column;
+      align-self: center;
+    }
+
+    #txt-area.box:hover,
+    #txt-area.box:focus {
+      color: #e27c57;
+      outline: none !important;
+      border-color: #4dbba6;
+      box-shadow: 0 0 10px #4dbba6;
+    }
+
+    ::selection {
+      color: white;
+      background-color: #e27c57;
+    }
+
+    .bttn-3 {
+      box-shadow: 0.4rem 0.25rem 0.25rem #00000040,
+        0.188rem 0.188rem 0.188rem #00000052;
+      background-color: #dd4d19;
+      color: #ffffff;
+      font-family: SpecialElite;
+      font-size: 1.25rem;
+      font-weight: 20;
+      text-shadow: 0.313rem 0.313rem 0.313rem #00000050;
+      width: 10rem;
+      height: 4.063rem;
+      border-color: transparent;
+      border-radius: 0.063rem;
+      position: relative;
+      top: 0.625rem;
+      margin-top: 3rem;
+      align-self: center;
+    }
+
+    #citation {
+      font-family: SpecialElite;
+      font-size: 0.9rem;
+      color: #e27c57;
+      text-align: center;
+      margin-top: 3rem;
+      margin-bottom: 0.5rem;
+    }
+
+    #bell-hooks {
+      font-family: SpecialElite;
```

Comment:
I'm not 100% sure but I think you can write the font-family once and everything on top of it will inherit it automatically

Reply:
Totally right! Just fixed this! Thanks :sweat_smile: